app/views/teachers/schools/show.html.erb

```diff
@@ -28,7 +28,7 @@
   </div>
   <div class="card border-0">
   </div>
-    <div class="card border-0">
+    <div class="card border-0 d-block d-sm-none d-md-block">
```

Comment:
Hide this when view port shrinks - we switch to three columns and show it further down instead.